src/test/java/mobile/TestOnlinerSearch.java

```diff
@@ -0,0 +1,38 @@
+package mobile;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import testing_of_mobile.driver.DriverSetup;
+import testing_of_mobile.FunctionForMobile;
+
+import java.net.MalformedURLException;
+
+public class TestOnlinerSearch extends FunctionForMobile {
+
+    @BeforeTest
+    public void setup() {
+        try {
+            new DriverSetup().getDriver("onliner");
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @AfterTest
+    public void closeDriver() {
+        killDriver();
+    }
+
+    @Test(description = "Check that the firs Iphone 8 plus costs 1350 p")
+    public void searchInOnliner() {
+        String idLocator = "by.onliner.catalog:id/";
```

Comment:
Why does it look like a test script from the first WD module (without POs, a wrapper for the driver, etc...)?